iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/ProcedureManager.java

```diff
@@ -1017,25 +951,15 @@ && new UpdateProcedurePlan(createTriggerProcedure).getSerializedSize() > planSiz
    *     {@link TSStatusCode#DROP_TRIGGER_ERROR} otherwise
    */
   public TSStatus dropTrigger(String triggerName, boolean isGeneratedByPipe) {
-    long procedureId =
-        executor.submitProcedure(new DropTriggerProcedure(triggerName, isGeneratedByPipe));
-    final List<TSStatus> statusList = new ArrayList<>();
-    final boolean isSucceed =
-        waitingProcedureFinished(Collections.singletonList(procedureId), statusList);
-    if (isSucceed) {
-      return RpcUtils.SUCCESS_STATUS;
-    } else {
-      return new TSStatus(TSStatusCode.DROP_TRIGGER_ERROR.getStatusCode())
```

Comment:
Seemingly in functions like this, the return status for failure may change.. This may affect IT and Cli/workbench.

Reply:
Fixed